src/index.css

```diff
@@ -0,0 +1,69 @@
+/** @format */
+
+body {
+  margin: 0;
+  background-color: rgb(241, 67, 96) !important;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white !important;
+  font-weight: bold !important;
+  padding: 0 10%;
+}
+
+nav {
+  font-size: 2.5rem;
+  margin-bottom: 5%;
+  display: flex;
+  flex-direction: column;
+}
+
+input {
+  width: 50%;
+  border-radius: 5px;
+}
+
+a {
+  text-decoration: none !important;
+  color: white !important;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+.overall-container {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.text-center {
+  text-align: center;
+}
+
+#item-container {
+  display: flex;
+  gap: 5px;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 2px 4px rgba(0, 0, 0, 0.06);
+  padding: 10px;
+}
+
+.item-details {
+  display: flex;
+  flex-direction: row;
+  gap: 5%;
+}
+
+.item-details-div {
+  margin-top: 10%;
+}
+
+.itemLink {
+  margin-bottom: 10%;
+}
+
+img {
+  width: 150px;
+  height: 150px;
+}
+
+@media all and (max-width: 768px) {
+  .overall-container {
+    grid-template-columns: repeat(1, minmax(0, 1fr));
+  }
```

Comment:
- [ ] Good job so far, Kindly note, however, we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Let's therefore ensure that the cards are displaying 2 on a row on mobile screens and follow the alternating color pattern on the homepage similar to the [original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Difference between background color of each card and the cards opposite on horizontal and vertical sides should be lighter and darker theme of the chosen color, like seen in the original design.


**Your design:**        |  **The origin design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Your desgin" src="https://github.com/Dorcas126/capstone-project-metrics-webapp/assets/12745474/a4301a70-0a60-4209-bb23-f9310bc21872"> |  <img width="800" alt="Home Page" src="https://github.com/Dorcas126/capstone-project-metrics-webapp/assets/12745474/e91e9d83-4842-4934-b961-044e3dab00ed">